web/src/containers/activity/EntryDetails.js

```diff
@@ -79,13 +79,13 @@ const EntryDetails = ({ activityIndex, fundingSource, activityKey, name }) => {
       >
         <Review heading={title} headingLevel={4} editContent={editContent} />
         <div className={collapsed ? 'visibility--print' : ''}>
-          <Overview activityIndex={activityIndex} />
-          <Goals aKey={activityKey} />
-          <Schedule aKey={activityKey} />
-          <Costs aKey={activityKey} />
-          <ContractorResources activityIndex={activityIndex} />
-          <CostAllocate aKey={activityKey} />
-          <StandardsAndConditions aKey={activityKey} />
+          <Overview activityIndex={index} />
+          <Goals aKey={activity.key} />
```

Comment:
It looks like `EntryDetails` doesn't actually take a whole activity anywhere, just the name and funding source, so `activity` isn't defined. We can use the `activityKey` prop instead.

```suggestion
          <Goals aKey={activityKey} />
```